packages/node/src/node/server/ProtocolService.ts

```diff
@@ -299,6 +297,30 @@ class ClusterState implements ClusterProtocol {
         this.type = type;
         this.#datasource = backing.datasource;
         this.#endpointId = backing.endpoint.number;
+
+        const attributeNameToIdMap = new Map<string, AttributeId>();
+        // For quieter attributes, we need to use the online events to get real state changes
+        for (const attr of type.attributes) {
+            attributeNameToIdMap.set(attr.name, attr.id);
+            if (attr.quieter) {
+                (this.#datasource.events[`${attr.name}$Changed`] as unknown as QuietEvent).online.on(() =>
```

Comment:
For these events (and the `stateChanged` one below) we should probably register them using an `ObserverGroup`...  So we can do `this.#observers.close()` to unregister them in `deleteCluster`

Reply:
Yes had on Todo. Done